compiler/plc_index/src/lib.rs

```diff
@@ -64,6 +64,11 @@ impl GlobalContext {
         &code.source[span]
     }
 
+    pub fn slice_trimmed(&self, location: &SourceLocation) -> String {
+        let slice = self.slice(location);
+        slice.split_whitespace().collect::<Vec<_>>().join(" ")
+    }
```

Comment:
I'm wondering, should this be the default behavior i.e. replacing the `slice` method altogether or leave it as is right now.